backend/endpoints/rooms.py

```diff
@@ -353,3 +353,92 @@ async def close_conversation(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to close conversation",
     )
+
+
+@router.get(
+    "/org/{org_id}/rooms/{room_id}/members",
+    response_model=ResponseModel,
+    status_code=status.HTTP_200_OK,
+    responses={
+        400: {"detail": "Invalid keyword supplied"},
+        404: {"detail": "Room not found"},
+        424: {"detail": "Failure to retrieve room members"},
+    },
+)
+async def get_members(org_id: str, room_id: str, keyword: Optional[str] = None):
```

Comment:
Can this keyword be role? 
@pauline-banye

Reply:
Yes it can, I deliberately didn't use role so its not mixed up with the member["role"] but if it's an ish, I can adjust it.